File: python/meteroid/models/json_config_value.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel


@dataclasses.dataclass
class JsonConfigValue(BaseModel):
    """A structured (JSON) config value — the "metadata" case, several fields in one entitlement."""

    value: t.Dict[str, t.Any]

File: browser/src/models/jsonConfigValue.ts
// this file is @generated
/** A structured (JSON) config value — the "metadata" case, several fields in one entitlement. */
export interface JsonConfigValue {
  value: unknown;
}

export const JsonConfigValueSerializer = {
  _fromJsonObject(object: any): JsonConfigValue {
    return {
      value: object["value"],
    };
  },

  _toJsonObject(self: JsonConfigValue): any {
    return {
      value: self.value,
    };
  },
};
